packages/grant-explorer/src/features/discovery/ExploreProjectsPage.tsx

```diff
@@ -19,7 +18,7 @@ import { CollectionDetails } from "../collections/CollectionDetails";
 import { FilterDropdown, FilterDropdownOption } from "../common/FilterDropdown";
 import { allChains } from "../../app/chainConfig";
 
-type Filter =
+export type Filter =
```

Comment:
this filter is defined twice, once here and in `useApplications`, are they the same thing?

Reply:
Thanks. Updated!